### PR TITLE
feat(client): :sparkles: Show hands in lobby

### PR DESCRIPTION
**Pull request by zarik5**
Controllers are rendered as 3d crossairs, hands are rendered as wireframe skeleton.

This is the last requirement for AppLab

### CODE REVIEW

File: alvr/client_core/src/graphics/lobby.rs [hunk 1]
```diff
@@ -169,8 +192,11 @@ impl LobbyRenderer {
         }
     }
 
-    #[allow(unused_variables)]
-    pub fn render(&self, view_inputs: [RenderViewInput; 2]) {
+    pub fn render(
+        &self,
+        view_inputs: [RenderViewInput; 2],
+        hand_poses: [(Option<Pose>, Option<[Pose; 26]>); 2],
```
Comment (The-personified-devil) [inline alvr/client_core/src/graphics/lobby.rs:198]: This should only really concern itself with rendering, so it should take a precomputed list of positions and the computation part should be moved into calling code

File: alvr/client_openxr/src/lobby.rs [hunk 2]
```diff
@@ -95,6 +99,24 @@ impl Lobby {
             vec![crate::default_view(), crate::default_view()]
         };
 
+        self.xr_session
```
Comment (The-personified-devil) [inline alvr/client_openxr/src/lobby.rs:102]: Can't the device potentially return hand data when the hand is gripping the controller (just based on position relative to the controller) or is that not an issue?

**Review by The-personified-devil — APPROVED** [review]

File: alvr/client_openxr/src/lobby.rs (hunk 2 above)
Comment (zarik5) [reply alvr/client_openxr/src/lobby.rs:102]: What you are talking about is hybrid controllers and currently not supported but requested. Currently the implementation of `get_hand_motion()` always returns a global hand/controller pose if hand tracking data is present, using the palm joint.

**Comment by junlinp** [comment]
I wonder how to enable this?

**Pull request merged**